src/user/announcements.rst

```diff
@@ -8,6 +8,21 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 2023
 ----
 
+2023-08-xx: Dropping support for MacOS <10.13
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Conda-forge tries to support OS versions as long as possible, often far beyond the support of the
+OS vendor. For the last seven years, we have supported a minimal MacOS version of 10.9, despite
+the fact that Apple has released new versions on a yearly basis, and aggressively pushes its users
+to upgrade. One big reason that we have been able to do this at all, is that conda-forge is able
+to ship an up-to-date C++ standard library in its environments, but we have now reached the end of
+the road, as several key projects in the ecosystem are breaking (despite this measure) when
+compiling against such old versions of the SDK.
+
+As most things in conda-forge, support for such old versions happens on a best-effort basis, and
+we reserve the right to drop support for old versions as it becomes necessary for various reasons.
```

Comment:
I think this is fine to mention, but the legalistic language might be a bit off-putting for a community project. We could write something like: "... and unfortunately there comes a point when it is no longer practical to support certain systems."